qiskit/quantum_info/__init__.py

```diff
@@ -5,11 +5,12 @@
 # This source code is licensed under the Apache License, Version 2.0 found in
 # the LICENSE.txt file in the root directory of this source tree.
 
-"""Quantum Information  methods."""
+"""Quantum Information methods."""
 
 from .operators.unitary import Unitary
 from .operators.pauli import Pauli, pauli_group
 from .operators.channel import Choi, SuperOp, Kraus, Stinespring, Chi, PTM
-from .states.states import basis_state, random_state, projector, purity
-from .states.measures import state_fidelity
 from .operators.measures import process_fidelity
+from .states.states import basis_state, projector, purity
+from .states.measures import state_fidelity
+from .random import random_unitary, random_state, random_density_matrix
```

Comment:
Should these be top level or instead things like `from . import random` so that you have to type `qi.random.random_unitary` to use these functions

Reply:
I think the `.random` file is also missing from your PR

Reply:
right now they are accessible in both ways. i thought it's more convenient and we have a bunch of other things at the top level of this package, so it seems arbitrary to exclude these.
```
from qiskit.quantum_info import random_unitary
from qiskit.quantum_info.random import random_unitary
```

Reply:
I didn't mean just random, but also the other things like projector, purity etc. While it's not many functions now this will grow a lot soon so we should start trying to make it well organised now rather than changing import paths later which would then break any existing code that uses top level imports.

I think we should do this in a seperate PR though rather than trying to cram too much into one.

Reply:
i agree